tests/test_isrTaskLSST.py

```diff
@@ -337,7 +347,14 @@ def test_isrBias(self):
         self.assertLess(np.std(result.exposure.image.array[good_pixels]), 2.0*mock_config.readNoise)
 
         delta = result2.exposure.image.array - result.exposure.image.array
-        self.assertFloatsAlmostEqual(delta[good_pixels], self.bias.image.array[good_pixels], atol=1e-5)
+
+        # Note that the bias is made with bias noise + read noise, and
+        # the image contains read noise.
```

Comment:
I don't know why this was changed.  The delta here is between the processing where we applied the bias, and where we didn't apply the bias, but with the same read noise pattern.